index.ts

```diff
@@ -62,17 +62,17 @@ function handleSerialization(input: string, api_call: string): string {
 }
 
 export function debug(message: string): void {
-  return handleSerialization(message, "hc_debug");
+  handleSerialization(message, (e: u32): u32 => env.hc_debug(e));
```

Comment:
For some reason you can't just pass env.hc_debug it has to be wrapped in an inline function. It's kinda messy but maybe better than all that duplication

Reply:
I thought you couldn't pass functions as params at all, so despite the oddities, this is still good news